src/main/java/org/onebusaway/alexa/util/SpeechUtil.java

```diff
@@ -251,4 +139,22 @@ public static String getAnnounceFeaturev1_1_0Text(Session session) {
     public static String formatRegionName(String regionName) {
         return regionName.replace(" (beta)", "");
     }
+
+    /**
+     * Util to replace all special characters with whitespace. Stop name will contains spacial character such as '&',
```

Comment:
It's common for stops to be named `131st Street & 2nd Avenue`. Shouldn't the `&` be replaced with `and` to make this understandable to the user? Otherwise it would read `131st Street 2nd Avenue`?

Reply:
Similarly, I've also seen `131st Street + 2nd Avenue`.

Reply:
And, `131st Street @ 2nd Avenue`

Reply:
I can add special case for `@, & and +`, and remove other symbols, thanks

Reply:
Thanks - to clarify, here's the mapping:
* `&` -> `and`
* `+` -> `and`
* `@` -> `at`

Reply:
Thanks, added